### ISSUE TITLE
After today's Mac OSX update .test resolution stopped working.

### ISSUE THREAD

**Issue by keenminded** [weprovide/valet-plus#232]
Hello everyone,
Happy Friday!
After updating Mac OSX High Sierra to the latest version this morning, the .test local domain resolution stopped working. Is anyone else experiencing the same problem?
**Symptoms:** web browser just spins without connecting, cannot even connect to mailhog.test.
valet paths command is displaying the original configured path. Tried resetting it again, tried valet fix all with no avail.
Updated valet to the latest version, no errors are displayed during the installation. Trying to use php 7.1. Any ideas would be greatly appreciated. Would really like to get back on track here...

**Comment by samgranger** [comment]
Upgrading my system now to try and duplicate this issue. For now, could you tell me the output when you run `brew services ls`? Is dnsmasq running? If so, can you try and configure a different extension instead of `.test` for now?

**Comment by keenminded** [comment]
@samgranger Thank you for jumping in on this! My brew services list returns:
```Name    Status  User Plist
dnsmasq started root /Library/LaunchDaemons/homebrew.mxcl.dnsmasq.plist
mailhog started root /Users/pegasus/Library/LaunchAgents/homebrew.mxcl.mailhog.plist
mariadb started root /Users/pegasus/Library/LaunchAgents/homebrew.mxcl.mariadb.plist
nginx   started root /Library/LaunchDaemons/homebrew.mxcl.nginx.plist
php@7.1 started root /Library/LaunchDaemons/homebrew.mxcl.php@7.1.plist
redis   started root /Users/pegasus/Library/LaunchAgents/homebrew.mxcl.redis.plist
```
I am still unable to ping any of my local .test domains. Is there anything I need to check on?
I have restarted several times just to make sure. Nothing seems to take so far. No errors either, just times out. It's really bizarre since it was all working just this morning right before I ran the update.

**Comment by keenminded** [comment]
@samgranger Thank you for pointing me in the right direction. In the end, it was a line in the dnsmasq configuration file pointing to a no-longer-existing user profile .config, which took effect after the reboot of the system, and therefore no longer worked. After removing the offending line and restarting all services everything seems back to normal. Thanks again for all the help!

**Comment by samgranger** [comment]
Glad you managed to get this issue resolved & thank you for letting me know what the issue was/how it was resolved - hopefully nobody else will encounter it but if they do, now I know why!

**Comment by keenminded** [comment]
Of course! This large knowledge base is what keeps us all going.

Just so you know some additional specifics, and I haven't had the time to figure out a way around it, the issue was primarily with my setup. I had installed valet through composer globally. However, the process assigned my primary user as the owner of the binary files, and when I tried to run valet through a new user profile on the machine, there were all sorts of problems with permissions.

In the end, it started to seem to me that installing and running valet locally for each user on the system would be a cleaner and a simpler approach than having it running globally for multiple profiles.

Hope this helps, and thanks for all the hard work. Kudos!
